### ISSUE TITLE
Issue with generating Excel file

### ISSUE THREAD

**Issue by igigiberia** [EPPlusSoftware/EPPlus#397]
After generating a file, when I open it and close without any modification, it asks to save this file every time.
Please see attached file as an example.
[EmsIsufNoTakinRep_202105041843_EmsIsufNoTakin.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/6535085/EmsIsufNoTakinRep_202105041843_EmsIsufNoTakin.xlsx)

**Comment by JanKallman** [comment]
Yes, EPPlus currently don't create the pivot table records and the actual table, so Excel will generate it on opening the workbook. I assume this is why Excel asks to save the workbook. I know Microsoft has removed a lot of these "Save" messages with untouched workbooks, but apparently in this case with pivot tables. We will look into creating the pivot table records in future versions, when we add calculation for pivot tables and the GETPIVOTDATA method.

**Comment by akantro** [comment]
Is there a separate issue in github that is tracking "fully implement support for pivot table calculation" that i can add as a watch item?
